src/relay/pass/fuse_ops.cc

```diff
@@ -863,7 +865,7 @@ class FuseMutator : private ExprMutator {
     auto* ret_group = gmap_.at(tuple_get)->FindRoot();
     auto new_tuple = GetNewArguments({tuple_get->tuple}, ret_group)[0];
     auto new_node = TupleGetItemNode::make(new_tuple, tuple_get->index);
-    if (ret_group == gmap_.at(tuple_get)) {
+    if (ret_group->root_ref == tuple_get) {
```

Comment:
why do we need this change? Aren't these conditions equivalent?

Reply:
this is only due to consistency to the same case on Call.

Reply:
(a small refactoring)

Reply:
I see, then you can also update [the same condition in Tuple](https://github.com/dmlc/tvm/blob/master/src/relay/pass/fuse_ops.cc#L854) too.

Reply:
good catch.

Reply:
done.